python/src/DistributionImplementation_doc.i.in

```diff
@@ -1903,7 +1917,7 @@ Linv : :class:`~openturns.TriangularMatrix`
 
 See also
 --------
-getCholesky"
+openturns.Distribution.getCholesky"
```

Comment:
I don't understand why the change is necessary here. The link already works.

Reply:
I think its because of the distribution in experimental where inherited docstrings dont have the same module reference so they become invalid for these particular distributions